e2e/cypress/integration/account_settings/display/timezone_display_mode_spec.js

```diff
@@ -70,10 +70,10 @@ describe('Account Settings > Display > Timezone Mode', () => {
             {
```

Comment:
I missed to comment on your previous PR about resetting timezone at line 44. It's done already on global before hook. Does it still need here?

Reply:
Quick question: Is the global before hook being invoked once per suite run or once per test file run? If it's being invoked only once during the whole run then I need to reset, otherwise I'll remove.

Reply:
We're running per test file - https://github.com/mattermost/mattermost-webapp/blob/master/e2e/run_tests.js#L88

Reply:
Updated